.eslintrc.js

```diff
@@ -1,64 +1,22 @@
 module.exports = {
 	env: {
-		es6: true, // sets the 'ecmaVersion' parser option to 6
 		node: true
 	},
-	extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+	extends: ['enact', 'plugin:prettier/recommended', 'prettier/babel', 'prettier/react'],
```

Comment:
Should these extend from `enact/strict`?

Reply:
Can if you want, though I didn't on CLI nor dev-utils since the stylistic aspects are already handled by prettier, so I didn't feel that was necessary.